theano/sparse/tests/test_basic.py

```diff
@@ -2965,7 +2965,8 @@ def structured_function(*args):
     sparse.trunc,
     np.trunc,
     test_dtypes=[m for m in sparse.all_dtypes
-                 if not m in sparse.complex_dtypes])
+                 if not m in sparse.complex_dtypes],
+    grad_test=False)
```

Comment:
I don't think this is correct, we should actually check it gives 0 most of the time.
I have a fix in #5693 that avoid sampling close to the places it would be evaluated to non-zero.